chiboycalix/andela-banka-frontend
# #165800700 Staff can delete account

#### What does this PR do?
          - Delete a User bank account
#### Description of Task to be completed?
         - Write functions to perform the delete operation
#### How should this be manually tested?
         - By entering the required url in a browser
         - Log in as a staff (email: staff@gmail.com, password: staff)
         - navigate to the delete account page
         - you will see a delete button
         - click on it to perform the delete operation
#### Any background context you want to provide?
         - none
#### What are the relevant pivotal tracker stories?
         - [#165800700](https://www.pivotaltracker.com/story/show/165800700)
#### Screenshots (if appropriate)
         
![Screenshot (26)](https://user-images.githubusercontent.com/35861980/57178085-6feafe80-6e63-11e9-9885-37d60267df0c.png)

#### Questions:
         - none
